nodes/pods/nodes-pods-user-namespaces.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: nodes-pods-user-namespaces
+[id="nodes-pods-user-namespaces"]
+= Running pods in Linux user namespaces
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+Linux user namespaces allow administrators the ability to isolate the container user and group identifiers (UIDs and GIDs) so that a container can have a different set of permissions than the host system it is running on. For example, the container can be granted full privileges for operations inside the user namespace, but can be unprivileged for operations outside the namespace.
+
+By default, a continer runs in the host system's user namespace. Running a container in the host user namespace can be useful when the container needs a feature that is available only in the host user namespace. However, it introduces security concerns, such as the possibility of container breakouts, where containers can run as root on the host without actually having root privileges on the host. 
```

Comment:
> where containers can run as root on the host without actually having root privileges on the host.

this is the opposite. The risk of running pods in the root user namespace is the UID inside the container is the same as on the outside, and so if a container breaks out, it will be treated as a user on the host. User namespaces mitigate this